src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+msgstr "Chatinstellingen"
+
+#: src/components/dms/ConvoMenu.tsx:84
+msgid "Chat unmuted"
+msgstr "Chat niet meer genegeerd"
+
+#: src/screens/SignupQueued.tsx:78
+#: src/screens/SignupQueued.tsx:82
+msgid "Check my status"
+msgstr "Controleer mijn status"
+
+#: src/screens/Login/LoginForm.tsx:275
+msgid "Check your email for a login code and enter it here."
+msgstr "Controleer uw e-mail voor een aanmeldcode en voer deze hier in."
+
+#: src/view/com/modals/DeleteAccount.tsx:231
+msgid "Check your inbox for an email with the confirmation code to enter below:"
+msgstr "Controleer uw inbox voor een e-mail met de bevestigingscode om hieronder in te voeren:"
+
+#: src/screens/StarterPack/Wizard/index.tsx:199
+msgid "Choose Feeds"
+msgstr "Kies Feeds"
+
+#: src/screens/StarterPack/Wizard/index.tsx:195
+msgid "Choose People"
+msgstr "Kies mensen"
+
+#: src/view/com/auth/server-input/index.tsx:76
+msgid "Choose Service"
+msgstr "Kies dienst"
+
+#: src/components/StarterPack/ProfileStarterPacks.tsx:291
+msgid "Choose for me"
+msgstr "Kies voor mij"
+
+#: src/view/com/composer/labels/LabelsBtn.tsx:116
+msgid "Choose self-labels that are applicable for the media you are posting. If none are selected, this post is suitable for all audiences."
+msgstr "Kies labels die van toepassing zijn op de media die u plaatst. Dit bericht is geschikt voor alle doelgroepen als er geen zijn geselecteerd."
+
+#: src/screens/Onboarding/StepFinished.tsx:271
+msgid "Choose the algorithms that power your custom feeds."
+msgstr "Kies de algoritmen die uw aangepaste feeds aansturen."
+
+#: src/screens/Onboarding/StepProfile/AvatarCreatorItems.tsx:107
+msgid "Choose this color as your avatar"
+msgstr "Kies deze kleur als uw avatar"
+
+#: src/screens/Signup/StepInfo/index.tsx:201
+msgid "Choose your password"
+msgstr "Kies uw wachtwoord"
+
+#: src/view/screens/Settings/index.tsx:877
+msgid "Clear all storage data"
+msgstr "Wis alle opslaggegevens"
+
+#: src/view/screens/Settings/index.tsx:880
+msgid "Clear all storage data (restart after this)"
+msgstr "Wis alle opslaggegevens (hierna opnieuw opstarten)"
+
+#: src/components/forms/SearchInput.tsx:70
+msgid "Clear search query"
+msgstr "Wis zoekopdracht"
+
+#: src/view/screens/Settings/index.tsx:878
+msgid "Clears all storage data"
+msgstr "Wist alle opslaggegevens"
+
+#: src/view/com/modals/DeleteAccount.tsx:216
+msgid "Click here for more information."
+msgstr "Klik hier voor meer informatie."
+
+#: src/view/com/modals/DeleteAccount.tsx:208
+msgid "Click here for more information on deactivating your account"
+msgstr "Klik hier voor meer informatie over het deactiveren van uw account"
+
+#: src/components/TagMenu/index.web.tsx:152
+msgid "Click here to open tag menu for {tag}"
+msgstr "Klik hier om het tagmenu voor {tag} te openen"
+
+#: src/components/dialogs/PostInteractionSettingsDialog.tsx:304
+msgid "Click to disable quote posts of this post."
+msgstr "Klik om citaatberichten voor dit bericht uit te schakelen."
```

Comment:
Whilst correct, it doesn't disable already quoted posts, if any, so "toestaan/verbieden" is probably a better translation.

```suggestion
msgstr "Klik om citaatberichten voor dit bericht te verbieden."
```